scheduler/src/cook/config.clj

```diff
@@ -58,7 +58,6 @@
                              "com.netflix.fenzo.TaskScheduler" :warn
                              "com.netflix.fenzo.AssignableVirtualMachine" :warn}}))
   ([{:keys [file] :or {file "log/cook.log"} {:keys [default] :or {default :info} :as overrides} :levels}]
-   (println "Initializing logging")
```

Comment:
Shouldn't be part of this PR :)

Reply:
It's related to this PR because I found it very annoying to run the unit tests with these `println`s, and I don't think they're necessary anyway